src/bodyComponent.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable import/no-cycle */
+
+import scores from './scores';
+import addScore from './addScore';
+
+async function component() {
```

Comment:
- [ ]  For this project, We expect you to follow ES6 syntax. You should refactor your functions to make them the ES6 arrow functions.

Example:
```javascript
const component = async () => { };
```